src/client/components/safe-copy-to-clipboard/safe-copy-to-clipboard.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
+ * Copyright 2017-2019 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import * as CopyToClipboard from "react-copy-to-clipboard";
+
+const copyOptions = { debug: false, message: "Copy to clipboard: #{key}, Enter", format: "text/plain" };
+
+interface SafeProps extends CopyToClipboard.Props {
```

Comment:
`type Props = Pick<CopyToClipboard.Props, “text” | “onCopy”>`

I’m not sure about children. Key is always available. We don’t use options right now so we shouldn’t allow it on our component.

Reply:
Any chance we could bump @types/react-copy-to-clipboard from 4.2.5 to 4.3.0? 4.2.5 is missing options.format and children in type. Then, something like this would work:

```
import * as React from "react";
import * as CopyToClipboard from "react-copy-to-clipboard";

type Props = Pick<CopyToClipboard.Props, "text" | "onCopy" | "children">

export function SafeCopyToClipboard(props: Props) {
  return <CopyToClipboard {...props} options={{format: "text/plain"}}>
      </CopyToClipboard>;
}
```

Reply:
Feel free to update the dependency